tools/vipps-checkout-api-v3-postman-collection.json

```diff
@@ -0,0 +1,231 @@
+              ],
+              "host": [
+                "{{base_url}}/checkout/v3"
+              ],
+              "query": [],
+              "variable": [
+                {
+                  "disabled": false,
+                  "type": "any",
+                  "value": "{{reference}}",
+                  "key": "reference",
+                  "description": "(Required) The reference of the session. Example: \"123123\"."
+                }
+              ]
+            },
+            "header": [
+              {
+                "disabled": false,
+                "key": "Vipps-System-Name",
+                "value": "{{vipps_system_name}}",
+                "description": ""
+              },
+              {
+                "disabled": false,
+                "key": "Vipps-System-Version",
+                "value": "{{vipps_system_version}}",
+                "description": ""
+              },
+              {
+                "disabled": false,
+                "key": "Vipps-System-Plugin-Name",
+                "value": "{{vipps_system_plugin_name}}",
+                "description": ""
+              },
+              {
+                "disabled": false,
+                "key": "Vipps-System-Plugin-Version",
+                "value": "{{vipps_system_plugin_version}}",
+                "description": ""
+              },
+              {
+                "disabled": false,
+                "key": "client_id",
+                "value": "{{client_id}}",
+                "description": ""
+              },
+              {
+                "disabled": false,
+                "key": "client_secret",
+                "value": "{{client_secret}}",
+                "description": ""
+              },
+              {
+                "disabled": false,
+                "key": "Ocp-Apim-Subscription-Key",
+                "value": "{{Ocp-Apim-Subscription-Key}}",
+                "description": ""
+              },
+              {
+                "disabled": false,
+                "key": "Merchant-Serial-Number",
+                "value": "{{merchantSerialNumber}}",
+                "description": ""
+              },
+              {
+                "key": "Accept",
+                "value": "application/json"
+              }
+            ],
+            "method": "GET",
+            "auth": null
+          },
+          "response": [],
+          "event": [],
+          "protocolProfileBehavior": {
+            "disableBodyPruning": true
+          }
+        }
+      ],
+      "event": []
+    }
+  ],
+  "event": [],
+  "info": {
+    "_postman_id": "2d622a69-3cd0-486e-addb-c6fe9e0264b6",
+    "name": "Checkout API v3 - Collection",
+    "schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+    "description": {
+      "content": "For details, see the [Checkout API Guide](https://vippsas.github.io/vipps-developer-docs/docs/APIs/checkout-api).",
+      "type": "text/plain"
+    }
+  }
+}
```

Comment:
EOF

Reply:
I think perhaps this is Windows ™️ 
Is there anything we can do automatically?